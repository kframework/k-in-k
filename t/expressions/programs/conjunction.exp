<exp>
  <k> True && ( False && True ) ~> .K </k>
</exp>
