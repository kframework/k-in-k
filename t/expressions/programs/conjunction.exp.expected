impCell{}(
    kCell{}(kseq{}(inj{BExpResult{}, KItem{}}(False{}()), dotk{}())),
    aCell{}(False{}())
)